plan/relations.go

```diff
@@ -194,6 +232,24 @@ func (v *VirtualTableReadRel) ToProtoPlanRel() *proto.PlanRel {
 	}
 }
 
+func (v *VirtualTableReadRel) Copy(newInputs ...Rel) (Rel, error) {
+	return v, nil
+}
+
+func (v *VirtualTableReadRel) CopyWithExpressionRewrite(rewriteFunc func(expr.Expression) (expr.Expression, error), newInputs ...Rel) (Rel, error) {
```

Comment:
for cases where we know that we're going to ignore / not use the `newInputs` parameter, you should use `_` as the var name to indicate that. 

i.e. 

```go
func (v *VirtualTableReadRel) CopyWithExpressionRewrite(rewriteFunc func(expr.Expression) (expr.Expression, error), _ ...Rel) (Rel, error) {
```